precise/parrot_neuron_ps.cpp

```diff
@@ -101,6 +101,7 @@ void
 parrot_neuron_ps::get_status( DictionaryDatum& d ) const
 {
   Archiving_Node::get_status( d );
+  def< bool >(d, names::is_precise_neuron, true);
```

Comment:
There should be no need to insert this manually. `parrot_neuron_ps::is_off_grid()` returns `true`, so the default mechanism should work.